demo_helper.py

```diff
@@ -0,0 +1,155 @@
+# Separate module for cross-correlating and making a demo plot
+# Author: A. Corstanje, (a.corstanje@astro.ru.nl), 2023
+
+import h5py
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import signal as scipy_signal
+
+def get_freq_axis(signal):
+    """
+    Return the frequency axis of a (real) FFT spectrum of time series 'signal' as 1D array
+    Frequencies in MHz (hence the 1.0e-6), assumed a 0.1 ns sample period
+    """
+    return (1.0e-6 * np.fft.rfftfreq(signal.shape[0], d=0.1e-9)) # MHz
```

Comment:
We could in future consider using the units class of NuRadioMC (which is an idea borrowed from Auger actually), which creates less unit confusion. But as stand-alone script with dependencies this is totally fine for now.

Reply:
Yes, that's a more fundamental choice. 
It gets the job done, currently, without too much confusion (I say MHz wherever needed).
Further compliance & integration is a worry for (a bit) later.